lua/astrocommunity/pack/typescript/init.lua

```diff
@@ -233,4 +233,13 @@ return {
       },
     },
   },
+  {
+    "nvim-neotest/neotest",
+    optional = true,
+    dependencies = { "nvim-neotest/neotest-jest" },
+    opts = function(_, opts)
+      if not opts.adapters then opts.adapters = {} end
+      table.insert(opts.adapters, require "neotest-jest" {})
```

Comment:
```suggestion
      table.insert(opts.adapters, require "neotest-jest"(require("astrocore").plugin_opts "neotest-jest"))
```

This allows the neotest adapter to be configured easily through it's `opts` table while still getting set up in `neotest`

Reply:
Nice, I didn't know about it.

Reply:
@mehalter I also added a documentation about how to add options in the `README`, because the plugin doesn't provide a `setup` function, so `lazy` throw an exception when trying to do something like

```lua
return {
  "nvim-neotest/neotest-jest",
  opts = function(_, opts)
    opts.jestCommand = "npm test --"
  end,
}
```

The workaround is simple, add a `config` key with an empty `function`. I don't know if there is a better approach in this case, tell me what do you think.